bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerHandle.java

```diff
@@ -1822,6 +1892,17 @@ void handleBookieFailure(final Map<Integer, BookieSocketAddress> failedBookies)
             return;
         }
 
+        if (writeFlags.contains(WriteFlag.DEFERRED_SYNC)) {
+            blockAddCompletions.decrementAndGet();
+            if (LOG.isDebugEnabled()) {
```

Comment:
The comment isn't aligned with the function call. Are you retrying or not?

Reply:
we are not retrying; so comment needs to be changed.

Reply:
done